src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -674,6 +826,42 @@ define(function(require) {
 			globalCallback();
 		},
 
+		/**************************************************
+		 *                  API requests                  *
+		 **************************************************/
+
+		/**
+		 * Lookups details for a list of numbers
+		 * @param  {Object} args
+		 * @param  {String[]} args.numbers Phone numbers to look up
+		 * @param  {Function} args.success  Success callback
+		 * @param  {Function} [args.error]  Error callback
+		 */
+		portWizardRequestPhoneNumbersLookup: function(args) {
+			var self = this;
+
+			monster.request({
+				resource: 'phonebook.lookupNumbers',
+				data: {
+					data: {
+						numbers: args.numbers
+					}
+				},
+				success: function(data) {
+					args.success(data.data);
+				},
+				error: function(data, error, globalHandler) {
+					var errorData = {
+						phonebookUnavailable: _.includes([0, 500], error.status)
+					};
+
+					_.has(args, 'error') && args.error(errorData);
+
+					globalHandler(data);
+				}
```

Comment:
Ah, my bad. I didn't realize your custom error callback was meant to handle every error.

In that case, you should make it mandatory, as it will always bypass `globalHandler`, something like:
```js
function(data, error) {
  args.error({
    isPhonebookUnavailable: _.includes([0, 500], error.status)
  });
}
```
I also took the opportunity to prepend `is` to the prop name, to denote the fact that it contains a boolean value, which later on makes is easier to use (`phonebookUnavailable` could contain the error payload for example).

Reply:
My bad too, that's right, the error callback should be mandatory if it will handle every error :sweat_smile: